pkg/container/docker_run.go

```diff
@@ -341,7 +344,7 @@ func (cr *containerReference) copyDir(dstPath string, srcPath string) common.Exe
 
 			sansPrefix := strings.TrimPrefix(file, srcPrefix)
 			split := strings.Split(sansPrefix, string(filepath.Separator))
-			if ignorer.Match(split, fi.IsDir()) {
+			if ignorer != nil && ignorer.Match(split, fi.IsDir()) {
```

Comment:
Maybe `useGitIgnore` instead `ignorer != nil` could be more clear. Just my opinion, the logic is the same (unless ignorer could be nil in some other case)